formsfeeder.client-cli/src/main/java/formsfeeder/client/cli/parameters/CommandLineAppParameters.java

```diff
@@ -22,9 +23,13 @@
 
 public class CommandLineAppParameters implements AppParameters {
 	private static final String HOST_LOCATION_SHORT_OPTION = "h";
-	private static final String HOST_LOCATION_LONG_OPTION = "host";	
+	private static final String HOST_LOCATION_LONG_OPTION = "host";
+	private static final String CONTEXT_ROOT_SHORT_OPTION = "c";
+	private static final String CONTEXT_ROOT_LONG_OPTION = "contextroot";
 	private static final String USER_CREDENTIALS_SHORT_OPTION = "u";
-	private static final String USER_CREDENTIALS_LONG_OPTION = "user";	
+	private static final String USER_CREDENTIALS_LONG_OPTION = "user";
+	private static final String HEADER_SHORT_OPTION = "header";

```

Comment:
So, everything looks great with just this one thing...  It's obviously a misnomer to call -header a "short" option.  Can we shorten it to "-hdr"?  That way it really is shorter but doesn't conflict with -h.  Sorry to be pedantic about it, but I think it's a small change and worth it to probably make it at this point.  If it was something internal only, I would accept it and change it later, but I'm worried that if we don't get this right, changing it later will seem arbitrary and disruptive.  I think we're better to get this right now before it comes gets entrenched by client code.

Reply:
Yeah no problem at all, I guess I should have thought longer, have definitely used hdr before for things. Was stuck on "single letter" short options in my thinking though.

I was also thinking about context root short option, should that be cr maybe? would it ever make sense to add correlation id as an option in the cli and maybe that one makes more sense as c for sort option?